parlai/scripts/eval_model.py

```diff
@@ -118,13 +118,27 @@ def _save_eval_stats(opt, report):
         f.write("\n")  # for jq
 
 
+def get_task_world_logs(task, world_logs, is_multitask=False):
+    if not is_multitask:
+        return world_logs
+    else:
+        base_outfile, extension = os.path.splitext(world_logs)
+        return f'{base_outfile}_{task}{extension}'
```

Comment:
`        return f'{base_outfile}_{task}.{extension}'
`

Reply:
No, splitext includes the . In the ext

Reply:
My bad, sorry!

Reply:
the extension contains "."